project/Dependencies.scala

```diff
@@ -22,7 +22,7 @@ object Dependencies {
   val scalaz = "org.scalaz" %% "scalaz-core" % "7.2.7"
   val selenium = "org.seleniumhq.selenium" % "selenium-java" % "3.14.0" % "test"
   val seleniumManager = "io.github.bonigarcia" % "webdrivermanager" % "2.2.5" % "test"
-  val specs2Extra = "org.specs2" %% "specs2-matcher-extra" % "3.8.6" % "test"
+  val specs2Extra = "org.specs2" %% "specs2-matcher-extra" % "4.5.1" % "test"
```

Comment:
this has to match the version in play to avoid a org.something.Json class not found exception
https://mvnrepository.com/artifact/com.typesafe.play/play-specs2_2.12/2.7.9
![image](https://user-images.githubusercontent.com/7304387/114203672-46d91f80-9950-11eb-8e54-032c89c82d8b.png)